### PR TITLE
Fixed typos

### PR DESCRIPTION
**Pull request by pborreli**

### CODE REVIEW

File: docs/man/git-lfs-clean.1.ronn [hunk 1]
```diff
@@ -11,7 +11,7 @@ Clean calculates a SHA-256 signature of the data from STDOUT, and outputs a
 Git LFS pointer file.  It also queues the file to be pushed by
 git-lfs-push(1).
 
-Clean is typicall run by Git's clean filter, configured by the repository's
+Clean is typical run by Git's clean filter, configured by the repository's
```
Comment (tclem) [inline docs/man/git-lfs-clean.1.ronn]: Maybe *typically*?

**Comment by tclem** [comment]
Hey thanks @pborreli! Just one comment and then I'll merge this in.

**Comment by pborreli** [comment]
done, thanks

**Comment by tclem** [comment]
:metal: Cheers!

**Pull request merged**